### ISSUE TITLE
Matrix-Matrix Multiplication Missing in Test Wrappers

### ISSUE THREAD

**Issue by alyfarahat** [uwhpsc-2016/homework2#4]
`mat_mat(A, B)` wrapper is missing in `wrappers.py`. I assume it should be there since we are not supposed to touch `wrappers.py`?

**Comment by mvelegar** [comment]
Yes you should not edit wrappers.py, I'll let @cswiercz address this one.

**Comment by cswiercz** [comment]
I am in the process of adding the wrapper.

**Comment by cswiercz** [comment]
Added now. Sending instructions on how to pull changes.
